graspy/embed/mds.py

```diff
@@ -203,6 +206,10 @@ def fit(self, X, y=None):
         self.components_ = U
         self.singular_values_ = D ** 0.5
         self.dissimilarity_matrix_ = dissimilarity_matrix
+        if X.ndim == 2:
```

Comment:
No need to check `X.ndim`. A single line `self.n_features_in_ == X.shape[1]` should do.